Library/Styles/Colors.swift

```diff
@@ -188,6 +188,11 @@ extension UIColor {
     return .hex(0x1A8CFF)
   }
 
+  // 0x2b60ff
+  public static var ksr_blue: UIColor {
```

Comment:
Did this get added through the colors script? I don't see it added to the `Colors.json`

Reply:
It's there. https://github.com/kickstarter/ios-oss/blob/feature-native-checkout/bin/ColorScript/Resources/Colors.json